### ISSUE TITLE
[Bug/Feature Request] Scope autocompletion

### ISSUE THREAD

**Issue by gander** [laravel-idea/plugin#383]
I use `barryvdh/laravel-ide-helper` to generate `phpDoc metadata` which adds scope functions correctly to `phpDoc` (`scopeFoo($query)` -> `foo()`), however when writing queries, the IDE tells me the `scopeFoo()` functions instead of `foo()`.

**Comment by adelf** [comment]
Yes. Currently, it can be fixed only by using Laravel Idea Eloquent helper code generation - https://laravel-idea.com/docs/eloquent

**Comment by gander** [comment]
Oh, I'll test it. I had a problem with the generator written by _Barry vd. Heuvel_: namely, it doesn't sort the parameters alphabetically, just a little bit so PHP will return him from the reflection function. As a result, every commit I had changes to phpDoc, even though nothing changed there. I didn't search too deeply, just at one point, just used my own sort function on `$this-> parameters`. However, did your solution provide for sorting or some other solution?

**Comment by adelf** [comment]
I have a lot of internal stuff in my helper code, so I only generate it in outside files...
